### PR TITLE
Use '{next}' where unambiguous

### PR DESCRIPTION
**Pull request by okurz**
Local verification run: http://lord.arch/tests/3341

### CODE REVIEW

File: tests/installation/addon_products_yast2.pm [hunk 1]
```diff
@@ -55,63 +55,63 @@ sub run() {
             else {
                 assert_screen "addon-license-$addon";
             }
-            send_key 'alt-a',                                       2;             # yes, agree
-            send_key 'alt-n',                                       2;             # next
+            send_key 'alt-a', 2;                                              # yes, agree
+            send_key $cmd{next}, 2;
```
Comment (dzedro) [inline tests/installation/addon_products_yast2.pm]: maybe fix also everytthin else than 1 ?

Comment (okurz) [inline tests/installation/addon_products_yast2.pm]: you mean I should also replace 'alt-a'?

**Comment by Soulofdestiny** [comment]
local verification? ;)

File: tests/installation/addon_products_yast2.pm (hunk 1 above)
Comment (dzedro) [inline tests/installation/addon_products_yast2.pm]: next is enough :+1:

**Comment by okurz** [comment]
Local verification run: http://lord.arch/tests/3341

**Pull request merged**